src/app/GitUI/UserControls/FileStatusList.Designer.cs

```diff
@@ -903,6 +904,14 @@ private void InitializeComponent()
             tsmiSaveAs.Text = "S&ave selected as...";
             tsmiSaveAs.Click += SaveAs_Click;
             // 
+            // tsmiMove
+            // 
+            tsmiMove.Image = Properties.Images.Renamed;
```

Comment:
Must be adapted to dark mode (not in Designer but you know that).
I am updating it to be ok in both modes, but for now it must be adapted.

The pr template that was updated to include scaling should maybe mention dark mode, win versions, ...